modules/index.d.ts

```diff
@@ -206,6 +206,11 @@ export interface BuilderProps {
   dispatch: Dispatch,
 }
 
+export interface ItemBuilderProps extends BuilderProps {
```

Comment:
`BuilderProps` is not suitable base type. 
Internal item props are not defined in type file.
Maybe define them (not all, just basic ones you can need) and use as base interface.
Or just not use base interface.